go/modelzooserver/modelzooserver.go

```diff
@@ -627,3 +667,51 @@ func (s *modelZooServer) DownloadModel(req *pb.ReleaseModelRequest, respStream p
 
 	return nil
 }
+
+func (s *modelZooServer) findLatestTag(name string) (string, error) {
```

Comment:
We may not need to find the latest model, let users specify the tag, or use an empty tag.